Marlin/src/module/configuration_store.cpp

```diff
@@ -1514,6 +1536,16 @@ void MarlinSettings::postprocess() {
         EEPROM_READ(servo_angles_arr);
       }
 
+      //
+      // Thermal first layer compensation values
+      //
+      #if ENABLED(USE_TEMP_COMPENSATION)
+        EEPROM_READ(temp_comp.z_offsets_probe);
+        EEPROM_READ(temp_comp.z_offsets_bed);
+        EEPROM_READ(temp_comp.z_offsets_ext);
+        temp_comp.reset_index();
+      #endif
```

Comment:
At the present time we write placeholder values to EEPROM for any disabled features, but in the future EEPROM data will be structured.